packages/ui/VirtualList/VirtualListBase.js

```diff
@@ -471,7 +471,7 @@ const VirtualListBaseFactory = (type) => {
 			const extent = Math.floor(index / dimensionToExtent);
 			let primaryPosition;
 
-			if (this.props.itemSizes) {
+			if (this.props.dataSize > index && this.props.itemSizes && typeof this.props.itemSizes[index] !== 'undefined') {
```

Comment:
Is it better to check `this.props.itemSizes` first because `VirtualList` usually does not include the `itemSize` prop, then the condition would be `false` quickly?

Reply:
👌